app/models/user.rb

```diff
@@ -33,17 +36,15 @@ def queued_video?(video)
   end
 
   private
+    def new_queue_item_review(queue_item, rating)
+      queue_item.video.reviews.create!(skip_validation: true, rating: rating, author: queue_item.user)
+    end
 
-  def new_queue_item_review(queue_item, rating)
-    queue_item.video.reviews.create!(skip_validation: true, rating: rating, author: queue_item.user)
-  end
-
-  def update_queue_item_rating(queue_item, rating)
-    queue_item.video.reviews.where(author: queue_item.user).first.update!(skip_validation: true, rating: rating)
-  end
-
-  def input_rating_empty?(rating)
-    rating == ""
-  end
+    def update_queue_item_rating(queue_item, rating)
+      queue_item.video.reviews.where(author: queue_item.user).first.update!(skip_validation: true, rating: rating)
```

Comment:
Be careful chaining this many methods together. If anything along the way fails, you have an unhandled failure which results in a 500 to your user. Usually you would want to grab the review, then check if it exists, then call update if so. That way you have the ability to handle the failure.